index.php

```diff
@@ -5,10 +5,12 @@
     include __DIR__.'/_templates/sitewide.php';
     include $template['header'];
     include $template['alert'];
+    require_once __DIR__.'/backend/config.loader.php';
     require_once __DIR__.'/backend/classify.current.php';
 ?>
             <script src="scripts/slider.js"></script>
             <script>var stripe_key = '<?php include __DIR__.'/backend/payment.php'; ?>';</script>
+            <script>var release = '<?=$config['release']?>';</script>
```

Comment:
I generally avoid shorthands like this for clarity, so I suppose we need to better define a code-style here. I'm sure we use `<?php echo $foo; ?>` elsewhere.